source/windows/host_resolver.c

```diff
@@ -35,6 +36,7 @@ int aws_default_dns_resolve(
     int res_error = GetAddrInfoA(hostname_cstr, NULL, &hints, &result);
 
     if (res_error) {
+        aws_win_log_message(AWS_LL_ERROR, AWS_LS_IO_DNS, "GetAddrInfoA()", res_error);
         AWS_LOGF_ERROR(AWS_LS_IO_DNS, "static: getaddrinfo failed with error_code %d", res_error);
```

Comment:
maybe just don't even bother with the 2nd LOGF statement. It's just a repeat of the new 1st one